docs/extensibility.md

```diff
@@ -27,129 +27,26 @@ wp.blocks.registerBlockType( 'mytheme/red-block', {
-	'core/paragraph',
-	'core/image',
-	'core/html',
-	'core/freeform'
-];
-
-wp.blocks.getBlockTypes().forEach( function( blockType ) {
-	if ( allowedBlocks.indexOf( blockType.name ) === -1 ) {
-		wp.blocks.unregisterBlockType( blockType.name );
-	}
-} );
-```
-
-## Hiding blocks from the inserter
-
-On the server, you can filter the list of blocks shown in the inserter using the `allowed_block_types` filter. you can return either true (all block types supported), false (no block types supported), or an array of block type names to allow.
-
-```php
-add_filter( 'allowed_block_types', function() {
-	return [ 'core/paragraph' ];
-} );
-```
-
-
-## Modifying Blocks (Experimental)
-
-To modify the behaviour of existing blocks, Gutenberg exposes a list of filters:
-
-#### `blocks.registerBlockType`
-
-Used to filter the block settings. It receives the block settings and the name of the block the registered block as arguments.
-
-#### `blocks.BlockEdit`
-
-Used to modify the block's `edit` component. It receives the original block `edit` component and returns a new wrapped component.
-
-#### `blocks.getBlockDefaultClassName`
-
-Generated HTML classes for blocks follow the `wp-block-{name}` nomenclature. This filter allows to provide an alternative class name.
-
-_Example:_
-
-```js
-// Our filter function
-function setBlockCustomClassName( className, blockName ) {
-	return blockName === 'core/code' ?
-		'my-plugin-code' :
-		className;
-}
-
-// Adding the filter
-wp.hooks.addFilter(
-	'blocks.getBlockDefaultClassName',
-	'my-plugin/set-block-custom-class-name',
-	setBlockCustomClassName
-);
-```
-
-#### `blocks.getSaveElement`
-
-A filter that applies to the result of a block's `save` function. This filter is used to replace or extend the element, for example using `wp.element.cloneElement` to modify the element's props or replace its children, or returning an entirely new element.
+## Extending the Editor UI
 
-#### `blocks.getSaveContent.extraProps`
- 
-A filter that applies to all blocks returning a WP Element in the `save` function. This filter is used to add extra props to the root element of the `save` function. For example: to add a className, an id, or any valid prop for this element. It receives the current props of the `save` element, the block type and the block attributes as arguments.
+Extending the editor UI can be accomplished with the `registerPlugin` API, allowing you to define all your plugin's UI elements in one place.
 
-_Example:_
+Refer to the [Plugins API](./extensibility/plugins-api) documentation for more information.
 
-Adding a background by default to all blocks.
+## Meta Boxes
 
-```js
-function addBackgroundColorStyle( props ) {
-	return Object.assign( props, { style: { backgroundColor: 'red' } } );
-}
-
-wp.hooks.addFilter(
-	'blocks.getSaveContent.extraProps',
-	'my-plugin/add-background-color-style',
-	addBackgroundColorStyle
-);
-```
+**It is highly encouraged converting PHP meta boxes to blocks!**
```

Comment:
I'm not certain about my English but should this be: "It is highly encouraged to convert PHP meta boxes to blocks!"

Reply:
Welcome to the club :)

I copied it over so no idea. I will wait for someone who is more experienced that two of us together :D

Reply:
> Converting PHP meta boxes to blocks is highly encouraged!

https://wordpress.slack.com/archives/C02QB2JS7/p1522311251000338